Resources/public/templates/fields/edit/relation.hbt

```diff
@@ -0,0 +1,37 @@
+<div class="pure-g ez-editfield-row">
+    <div class="pure-u ez-editfield-infos">
+        {{> ez_fieldinfo_tooltip }}
+        <label for="ez-field-{{ content.contentId }}-{{ fieldDefinition.identifier }}">
+            <p class="ez-fielddefinition-name">
+                {{ fieldDefinition.names.[eng-GB] }}{{#if isRequired}}*{{/if}}:
+            </p>
+            <p class="ez-editfield-error-message">&nbsp;</p>
+        </label>
+    </div>
+    <div class="pure-u ez-editfield-input-area ez-default-error-style">
+        {{#if isEmpty}}
+           <p>This field is empty</p>
+        {{else}}
+            {{#if destinationContent }}
+                <ul class="ez-relation-values">
+                     <li>"{{ destinationContent.name }}"</li>
+                     <li>Published: "{{ destinationContent.publishedDate }}"</li>
+                     <li>Last modification date: "{{ destinationContent.lastModificationDate }}"</li>
+                </ul>
+            {{else}}
+                {{#if loadingError}}
+                    <p class="ez-asynchronousview-error font-icon">
+                        An error occurred while loading the related content.
+                        <button class="ez-asynchronousview-retry ez-button font-icon pure-button">Retry</button>
+                    </p>
+                {{else}}
+                    <p>Loading the related content</p>
+                {{/if}}
+            {{/if}}
+        {{/if}}
+        <p>
+            <button class="ez-relationeditview-remove ez-button font-icon pure-button">Remove relation</button>
```

Comment:
instead of `ez-relationeditview-remove`, it should be something like `ez-relation-remove`. I would also put this button after the discover one.